src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,108 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
+@SuppressWarnings({"rawtypes"})
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75F;
+    private int index;
+    private int capacity;
+    private int size;
+    private Node[] nodes;
+
+    public MyHashMap() {
+        nodes = new Node[INITIAL_CAPACITY];
+        capacity = INITIAL_CAPACITY;
+    }
 
+    @SuppressWarnings({"unchecked"})
```

Comment:
This a bad practice i think

Reply:
This is taken from the original implementation of hashmap class. I prefer to keep it for now